Console/Command/ConfigListCommand.php

```diff
@@ -2,44 +2,76 @@
 namespace Thai\S3\Console\Command;
 
 use Magento\Config\Model\Config\Factory;
+use Magento\Framework\App\Area;
+use Magento\Framework\App\State;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class ConfigListCommand extends \Symfony\Component\Console\Command\Command
+/**
+ * @inheritdoc
+ */
+class ConfigListCommand extends Command
 {
     private $configFactory;
 
+    /**
+     * @var State
+     */
     private $state;
 
+    /**
+     * @param State $state
+     * @param Factory $configFactory
+     */
     public function __construct(
-        \Magento\Framework\App\State $state,
+        State $state,
         Factory $configFactory
     ) {
         $this->state = $state;
         $this->configFactory = $configFactory;
+
         parent::__construct();
     }
 
+    /**
+     * @inheritdoc
+     */
     protected function configure()
     {
         $this->setName('s3:config:list');
         $this->setDescription('Lists whatever credentials for S3 you have provided for Magento.');
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
I don't think we should use `@inheritdoc` here. The parent documentation is from Symfony about why the parent method is not abstract; it's not relevant to this implementation.

Reply:
It has return code descriptions and all `execute` commands are pretty same